.lagoon.yml

```diff
@@ -92,7 +104,19 @@ tasks:
           set -e
           drush en -y dpl_example_content
         service: cli
+    # @TODO How do we run this, even if something has gone wrong?
```

Comment:
I think you can use a [status check function](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#status-check-functions), probably `if: ${{ !cancelled() }}`